README.md

```diff
@@ -93,3 +93,15 @@ To build only the C++ library:
 % make test-out
 
 ```
+
+To build the C++ library with AVX512 enabling:
+```shell
+ENV parameter ORC_USER_SIMD_LEVEL is to switch "AVX512" and "NONE" at the running time.
```

Comment:
```suggestion
Environment variable ORC_USER_SIMD_LEVEL can be set to "AVX512" or "NONE" at compile and/or run time. At compile time, it defines the SIMD level to be compiled into the binaries. At run time, it defines the SIMD level to dispatch the code which can apply SIMD optimization. Note that if ORC_USER_SIMD_LEVEL is set to "NONE" at compile time, ORC_USER_SIMD_LEVEL will not take effect at run time even if it is set to "AVX512".
```

Reply:
Change to below one, please check it. Thank you very much!

Cmake option BUILD_ENABLE_AVX512 can be set to "ON" or (default value)"OFF" at the compile time. At compile time, it defines the SIMD level(AVX512) to be compiled into the binaries.
Environment variable ORC_USER_SIMD_LEVEL can be set to "AVX512" or (default value)"NONE" at the run time. At run time, it defines the SIMD level to dispatch the code which can apply SIMD optimization. 
Note that if ORC_USER_SIMD_LEVEL is set to "NONE" at run time, AVX512 will not take effect at run time even if BUILD_ENABLE_AVX512 is set to "ON" at compile time.